### PR TITLE
Added new drive support

### PR DESCRIPTION
**Pull request by Inba-Vinayagam**
Added a new drive support in micron plugin. Please review

### CODE REVIEW

File: plugins/micron/micron-nvme.c [hunk 1]
```diff
@@ -56,6 +56,7 @@ enum eDriveModel {
 	M51CX,
 	M5407,
 	M5411,
+  M6003,
```
Comment (igaw) [inline plugins/micron/micron-nvme.c:59]: The indention is off.

**Review by igaw — CHANGES_REQUESTED** [review]
Please use subject prefix, e.g. 'plugins/micron: add support for new drive' and also add a commit message and a Signed-off-by.

Also use the correct indention. Thanks

**Comment by igaw** [comment]
@hanumanthuh please also review. Thanks!

File: plugins/micron/micron-nvme.c (hunk 1 above)
Comment (Inba-Vinayagam) [reply plugins/micron/micron-nvme.c]: plugins/micron: fixed the indention issue and update the PR signed off by Inba Vinayagam

**Comment by igaw** [comment]
Alright, I'll close this PR for now. Please reopen or create a new one if you have an update. Thanks.

**Pull request closed without merging**